xmtp_mls/src/groups/sync.rs

```diff
@@ -64,8 +64,13 @@ where
     pub async fn sync(&self) -> Result<(), GroupError> {
         let conn = &mut self.client.store.conn()?;
         let provider = self.client.mls_provider(&conn);
-        self.add_missing_installations(provider).await?;
-        self.update_latest_installation_list_timestamp(conn).await?;
+
+        let (last, latest) = self.update_latest_installation_list_timestamp(conn).await?;
```

Comment:
We are essentially bumping the timestamp every time we sync, rather than only bumping the timestamp when we add missing installations. I think this is an issue - for example, if we sync every 15 minutes, then the timestamp will be bumped every time without ever triggering the missing installations to be added

Reply:
you are totally right.  fixed the logic.